examples/rbd/raw-block-pvc-rwop.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: raw-block-pvc
```

Comment:
this name is already used, please name it something else

Reply:
done

Reply:
I do still see the `raw-block-pvc` name here, and `rbd-pvc` in the `pvc-rwop.yaml` is already used too. Please use a name for the PVCs that contain `rwop` too.